src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i class="fa-regular fa-copy fa-xl" :class="{'fa-beat': animate, 'copied-icon': animate}" @click="copy"></i></div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "CopyToClipboard",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    animate: false
+  }),
+  methods: {
+    copy() {
+      javascript:navigator.clipboard.writeText(this.item.clipboard)
+      this.animate = true;
+      setTimeout(

```

Comment:
I think it shoud be possible to avoid the setTimeout by using CSS [`animation-delay`](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-delay), that would be great.

Reply:
I implemented it this way to use the font-awesome animation class. I can  remove this by including a custom animation in CSS. It's related to the other change about making the animation shorter.